guaccollect.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: page
+title: Ingesting data with GUACCollect
+permalink: /guaccollect/
+parent: GUAC demos
+grand_parent: Getting started with GUAC
+nav_order: 2
+---
+
+# Ingesting data with GUACCollect
+
+GUACCollect is a command-line tool within the GUAC ecosystem designed for collecting and ingesting software bill of materials (SBOMs), attestations, and other metadata documents from various sources. This tool supports a wide range of data sources, including GitHub, S3, Google Cloud Storage (GCS), and OCI images, making it a versatile choice for enhancing the visibility and security of your software supply chain.
+
+## Requirements
+
+- A fresh copy of the [GUAC service infrastructure through Docker Compose]({{
+  site.baseurl }}{%link setup.md %}). Including the `guacone` binary in your path
+  and [GUAC Data](https://github.com/guacsec/guac-data/archive/refs/heads/main.zip)
```

Comment:
hmm, i think we usually recommend cloning GUAC data instead of unzipping - is this a different dataset?